docs/userguide/objective.md

```diff
@@ -1,24 +1,77 @@
 # Objective
 
-## General Information
+The [`Objective`](baybe.objective.Objective) instructs BayBE how to deal with multiple
+targets (if applicable).
 
-Objectives allows you to effectively manage the optimization objective and are implemented by the [`Objective`](baybe.objective.Objective) class.
-By defining objectives, you can optimize a single target function or combine multiple target functions using different modes.
-To create an objective, it is necessary to provide at least the following:
-* **An optimization mode:** This can be either ``SINGLE`` or ``DESIRABILITY``, denoting either the optimization of a single target function or a combination of different target functions.
-* **A list of targets:** The list of targets that are optimized. Note that the ``SINGLE`` mode also requires a list containing the single target. See [here](./targets) for details.
+To create an objective, it is necessary to provide the following:
+* An optimization `mode`: This can be either `SINGLE` or `DESIRABILITY`,
+  denoting the optimization of a single target function or a combination of
+  different target functions respectively.
+* A list of `targets`: The list of targets that are optimized. Note that the
+  `SINGLE` mode also requires a list containing the single target.
 
-## Supported Optimization Modes
+```{note}
+We are actively working on adding more objective modes for multiple targets.
+```
 
+## Supported Optimization Modes
 Currently, BayBE offers two optimization modes.
 
-### Single Target Optimization: The ``SINGLE`` mode
+### SINGLE
+In the `SINGLE` mode, objectives focus on optimizing a single target. 
+Nearly all of the [examples](../../examples/examples) use this objective mode.
+
+```python
+from baybe.targets import NumericalTarget
+from baybe.objective import Objective
+
+target1 = NumericalTarget(name="yield", mode="MIN", bounds=(0, 100))
+objective = Objective(mode="SINGLE", targets=[target1])
+```
+
+### DESIRABILITY
+The ``DESIRABILITY`` mode enables you to combine multiple targets via scalarization 
+into a single value, a method well know and also utilized in classical DOE.
+
+Besides `mode` and `targets`, this objective type has two additional optional arguments:
+* `weights`: Some targets might be more important than others.
+  You can specify the relative weights of the targets in this argument.
```

Comment:
It feels a bit weird that you suddenly talk to the reader directly, while the other parts of this user guide are written in a more passive voice (same applies to the next bullet point). I'd prefer sometihng like "it is possible to specify.." and 
 then "BayBE automatically normalizes the provided numbers"

Reply:
I removed it in this objective guide as its not very necessary, but I think that style of writing is quite nice eg in cases of "If you want X, you can sue objectthisandthat" or "Suppose we now add thesandthat situation, we can do thisandthat..."

For instance it is still heavily used int he targets user guide

Is there any argument against it?

Reply:
In general, there is no argument against it. Here, it just felt weird because it felt like a sudden change in style within this user guide which is why I noted it here.